README.md

```diff
@@ -1,56 +1,51 @@
 ![](https://img.shields.io/badge/Microverse-blueviolet)
 
-# Project Name
-
-> Description the project.
+# Blog App
 
+> This would be a Blog app website as a project I am starting with on Rails. It would be functional Blog which does basic things all other blogs do such as comments, liking, posts, editing, deleting etc.
 
 ## Built With
 
-- Major languages
-- Frameworks
-- Technologies used
-
-## Live Demo (if available)
-
-[Live Demo Link](https://livedemo.com)
-
+- Ruby on Rails
+- CSS
 
 ## Getting Started
 
-**This is an example of how you may give instructions on setting up your project locally.**
-**Modify this file to match your project, remove sections that don't apply. For example: delete the testing section if the currect project doesn't require testing.**
-
-
 To get a local copy up and running follow these simple example steps.
 
 ### Prerequisites
 
 ### Setup
 
-### Install
+Cloning a repository
 
-### Usage
+- On GitHub.com, navigate to the main page of the repository;
 
-### Run tests
+- Above the list of files, click "Code" button;
 
-### Deployment
+- Copy the URL to clone the repository.
 
+  - To clone the repository run `https://github.com/Olamarx/Blog.git`
 
+- Open Terminal;
 
-## Authors
+- Change the current working directory to the location where you want the cloned directory;
+
+- Type `git clone`, and then paste the URL you copied earlier;
 
-👤 **Author1**
+### Install
+
+- To install necessary gem files in your machine run `bundle install` on your terminal.
+- To run your machine locally run `rails server` on your terminal.
```

Comment:
@Omar-Muhamad do you mean I should put in instructions of postgresql?

Reply:
I don't know what you mean by how to create database because even me, I created it manually with PG Admin. Is that what you mean I should add?

Reply:
Hi @Olamarx, I'm sure you know that running `bundle install` and then `rails server` the project won't run because we also need the database setup commands, so listing such commands was necessary.